dbt-snowflake/src/dbt/adapters/snowflake/impl.py

```diff
@@ -83,6 +84,24 @@ class SnowflakeAdapter(SQLAdapter):
         }
     )
 
+    def __init__(self, config, mp_context) -> None:
+        super().__init__(config, mp_context)
+        self.add_catalog_integration(constants.DEFAULT_NATIVE_CATALOG)
+        self.add_catalog_integration(constants.DEFAULT_ICEBERG_CATALOG)
+
+    def add_catalog_integration(
+        self, catalog_integration: CatalogIntegrationConfig
+    ) -> CatalogIntegration:
+        # Snowflake uppercases everything in their metadata tables
```

Comment:
this is not true actually. I have found exceptions where columns are lowercased, such as with the `is_iceberg` column.

Reply:
Wild, I could have sworn when I was troubleshooting this all the metadata came through as uppercase. Now it's all showing up as lowercase.